src/file_dialog.rs

```diff
@@ -1639,23 +1653,23 @@ impl FileDialog {
 
     /// Updates a device entry of a device list like "Devices" or "Removable Devices".
     fn ui_update_device_entry(&mut self, ui: &mut egui::Ui, device: &Disk) {
-        let label = match device.is_removable() {
-            true => format!(
-                "{}  {}",
+        let label = if device.is_removable() {
+            format!(
+                "{} {}",
                 self.config.removable_device_icon,
                 device.display_name()
-            ),
-            false => format!("{}  {}", self.config.device_icon, device.display_name()),
+            )
+        } else {
+            format!("{} {}", self.config.device_icon, device.display_name())
```

Comment:
These also have to be two spaces

```suggestion
            format!("{}  {}", self.config.device_icon, device.display_name())
```